typescript/agentkit/README.md

```diff
@@ -43,7 +43,8 @@ AgentKit is a framework for easily enabling AI agents to take actions onchain. I
 
 ## Getting Started
 
-*Prerequisites*:
```

Comment:
Please undo all these formatting changes to the README. The only change to this file should be the new table for the Flaunch actions